spacy/lang/lb/README.md

```diff
@@ -0,0 +1,12 @@
+Initial rules of basic NLP functions for Luxembourgish
+
+So far has been done:
+- basic tokenisation
+- stopwords
+- lemmatisation
+
+TODO:
+
+- tokenize cliticised definite article "d'" as token of its own: d'Kanner > [d'] [Kanner]
+- treat other apostrophes within words as part of the word: [op d'mannst], [fir d'éischt] (= exceptions)
+- norm execptions: find a possibility to deal with the zillions of spelling variants (vläicht = vlaicht, vleicht, viläicht, viläischt, etc. etc.)
```

Comment:
Thanks for writing this up, this is really helpful! 👍

I think it might be better to include these notes as a comment in the `lb/__init__.py` (or even within the files that they're referring to). This lets us avoid having arbitrary readmes and documentation files throughout the codebase.

Reply:
I wrote the necessary information into the respective files and deleted the readme.